src/frontends/tensorflow/include/openvino/frontend/tensorflow/frontend.hpp

```diff
@@ -62,6 +62,10 @@ class TENSORFLOW_API FrontEnd : public ov::frontend::FrontEnd {
     void add_extension(const std::shared_ptr<ov::Extension>& extension) override;
 
 protected:
+    /// \brief Runs middle transformations to convert sub-graphs with intermediate (frontend internal) operations
+    /// into sub-graphs with only OpenVINO operations
+    void apply_middle_transformations(const std::shared_ptr<ov::Model>& function) const;
```

Comment:
There is a special pass called normalize for this purpose. It is for running transformations after conversion to replace all unsupported ops.

Reply:
Moved to `normalize`